eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/io/DeprecatedVariantAccessionWriter.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.release.io;
+
+import org.springframework.batch.item.ExecutionContext;
+import org.springframework.batch.item.ItemStreamException;
+import org.springframework.batch.item.ItemStreamWriter;
+
+import uk.ac.ebi.eva.accession.core.persistence.DbsnpClusteredVariantOperationEntity;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+/**
+ * Writes the accessions of historical variants to a flat file.
+ */
+public class DeprecatedVariantAccessionWriter implements ItemStreamWriter<DbsnpClusteredVariantOperationEntity> {
+
+    private static final String FILE_SUFFIX = "_deprecated_ids.txt";
```

Comment:
Should we make it explicit with "_deprecated_rs_ids.txt"?

Reply:
well, will there be a "_deprecated_ss_ids.txt" file? if not, I'm ok with just "_deprecated_ids"

Reply:
We are only going to generate reports for RS IDs because non-submitters don't care about SS IDs.